core/data/azure/infrastructure/epiphany-single-machine/data.yaml

```diff
@@ -0,0 +1,601 @@
+#        #- name: "node1"
+#        #  address: "epiphany-vm1.domain.com"
+#        port: 30104
+
+  monitoring:
+    alerts:
+      enable: True # required value
+      handlers:
+        mail:
+          enable: False # required value
+          smtp_from: "alert@test.com"
+          smtp_host: "smtp-url:smtp-port"
+          smtp_auth_username: "your-smtp-user@domain.com"
+          smtp_auth_password: "your-smtp-password"
+          smtp_require_tls: True
+          recipients: 
+          - recipient1@domain.com
+          - recipient2@domain.com
+        slack:
+          enable: False # required value
+          api_url: url-to-slack-workspace-api.slack.com
+        pagerduty:
+          enable: False # required value
+          service_key: your-service-key 
+      rules:
+      - name: "UpDown"
+        expression: up == 0
+        duration: 1m #1s, 1m, 1h, 1d, 1w, ...
+        severity: critical
+        message: "Node is down." 
+      - name: "DiskSpace"
+        expression: ((node_filesystem_avail_bytes* 100) / node_filesystem_size_bytes) < 20 # 100 - 80
+        duration: 1m #1s, 1m, 1h, 1d, 1w, ...
+        severity: critical
+        message: "Disk usage is above 80%" 
+      - name: "DiskSpacePrediction"
+        expression: predict_linear(node_filesystem_free_bytes{job="node"}[1h], 48 * 3600) < 0
+        duration: 1m #1s, 1m, 1h, 1d, 1w, ...
+        severity: warning
+        message: "Disk will run out of space in less than 48h" 
+      - name: "MemoryUsage"
+        expression: (sum by (instance) (node_memory_MemTotal_bytes) - sum by (instance)(node_memory_MemFree_bytes + node_memory_Buffers_bytes + node_memory_Cached_bytes) ) / sum by (instance)(node_memory_MemTotal_bytes) * 100 > 80
+        duration: 15m #1s, 1m, 1h, 1d, 1w, ...
+        severity: warning
+        message: "Server memory has been used in more than 80% during last 15 minutes." 
+      - name: "CpuLoad"
+        expression: 100 - (avg by (instance) (irate(node_cpu_seconds_total{job="node",mode="idle"}[5m])) * 100) > 80
+        duration: 15m #1s, 1m, 1h, 1d, 1w, ...
+        severity: critical
+        message: "CPU utilization has exceeded 80% over last 15 minutes." 
+      - name: "KafkaConsumerLag"
+        expression: sum by(consumergroup) (kafka_consumergroup_lag) > 1000
+        duration: 15m #1s, 1m, 1h, 1d, 1w, ...
+        severity: critical
+        message: "Kafka consumers are lagging more than 1000 messages over last 15 minutes." 
+
+rabbitmq:
+  amqp_port: 5672 #optional - default 5672  
+  plugins: # optional list of RabbitMQ plugins
+    - rabbitmq_management
+    - rabbitmq_management_agent
+  policies: # optional list of RabbitMQ policies
+    - name: ha-policy2
+      pattern: ".*"
+      definitions:
+        ha-mode: all
+  custom_configurations: #optional list of RabbitMQ configurations (new format -> https://www.rabbitmq.com/configure.html)
+    - name: vm_memory_high_watermark.relative
```

Comment:
This part can be commented because we just want to show ability to create custom config - and do not require need this setting

Reply:
Fixed